src/api/java/baritone/api/utils/SettingsUtil.java

```diff
@@ -261,6 +262,36 @@ public String toString(ParserContext context, Object value) {
             public boolean accepts(Type type) {
                 return List.class.isAssignableFrom(TypeUtils.resolveBaseClass(type));
             }
+        },
+        MAPPING() {
+            @Override
+            public Object parse(ParserContext context, String raw) {
+                Type keyType = ((ParameterizedType) context.getSetting().getType()).getActualTypeArguments()[0];
```

Comment:
Is this confirmed to work in `standalone` where this is run through proguard? i.e. is this type information confirmed to be preserved?

Reply:
It does work, but the tab completion seems to break in `standalone` (on master as well), is that a known thing?